### PR TITLE
LOAD CELL: Remove force offset

### PR DESCRIPTION
**Pull request by StephanCoetzee**
Closes https://github.com/keyme/grbl/issues/96

I removed some trailing whitespace as well.

@Jeff-Ciesielski

### CODE REVIEW

File: signals.c [hunk 1]
```diff
@@ -46,16 +45,11 @@ void signals_update_force()
...
   // Index in signal buffer
   static const uint8_t n = N_FILTER - 1;
 
-  // Make sure that the offset doesn't make the sample negative.
-  if (signals.force_offset > raw_value) {
-    X_BUF(n) = 0;
-  } else {
-    X_BUF(n) = raw_value - signals.force_offset;
-  }
+  X_BUF(n) = raw_value;
```
Comment (Jeff-Ciesielski) [inline signals.c:52]: Should you still clamp this as >= 0?

**Review by Jeff-Ciesielski — COMMENTED** [review]
Minor concern.  Otherwise looks good.

File: signals.c (hunk 1 above)
Comment (StephanCoetzee) [reply signals.c:52]: Yes, that's a good idea.

Comment (StephanCoetzee) [reply signals.c:52]: No wait, it's not necessary. We cannot get a value below 0 anymore. raw_value is the value read from the ADC which will always be positive.

The subtraction of the offset made it possible for the value to be negative, but it's not longer the case.

**Comment by Jeff-Ciesielski** [comment]
LGTM

**Pull request merged**